### PR TITLE
[OPIK-417] Truncate base64 images for dataset items

### PR DESCRIPTION
**Pull request by idoberko2**
## Details
Similarly to traces and spans, we would like to truncate base64 encoded images in dataset items as well.

## Issues
OPIK-417

## Testing
Added a couple of E2E tests to cover the new flows.

### CODE REVIEW

**Review by thiagohora — DISMISSED** [review]
LGTM

File: apps/opik-backend/src/test/java/com/comet/opik/podam/manufacturer/JsonNodeTypeManufacturer.java [hunk 1]
```diff
@@ -9,20 +9,61 @@
...
+    private String getValidRandomString(int count) {
+        String randomString = RandomStringUtils.randomAlphanumeric(count);
+        while (!isValidString(randomString)) {
+            randomString = RandomStringUtils.randomAlphanumeric(count);
+        }
+
+        return randomString;
+    }
+
+    private boolean isValidString(String string) {
```
Comment (thiagohora) [inline apps/opik-backend/src/test/java/com/comet/opik/podam/manufacturer/JsonNodeTypeManufacturer.java]: Can we rename this param?

File: apps/opik-backend/src/test/java/com/comet/opik/podam/manufacturer/JsonNodeTypeManufacturer.java [hunk 2]
```diff
@@ -9,20 +9,61 @@
...
+        for (String forbiddenString : FORBIDDEN_STRINGS) {
+            if (string.startsWith(forbiddenString)) {
+                return false;
+            }
+        }
+
+        return true;
```
Comment (thiagohora) [inline apps/opik-backend/src/test/java/com/comet/opik/podam/manufacturer/JsonNodeTypeManufacturer.java]:
```suggestion
        return !FORBIDDEN_STRINGS.anyMatch(forbiddenString -> string.equals(forbiddenString))
```

File: apps/opik-backend/src/test/java/com/comet/opik/podam/manufacturer/JsonNodeTypeManufacturer.java (hunk 1 above)
Comment (idoberko2) [reply apps/opik-backend/src/test/java/com/comet/opik/podam/manufacturer/JsonNodeTypeManufacturer.java]: Removed this helper function

File: apps/opik-backend/src/test/java/com/comet/opik/podam/manufacturer/JsonNodeTypeManufacturer.java (hunk 2 above)
Comment (idoberko2) [reply apps/opik-backend/src/test/java/com/comet/opik/podam/manufacturer/JsonNodeTypeManufacturer.java]: Done in https://github.com/comet-ml/opik/pull/651/commits/dda862125339998cbbda4b6786dadba44312ab89

**Review by thiagohora — APPROVED** [review]

**Pull request merged**